README.md

```diff
@@ -1,16 +1,14 @@
-Simple Decorator for Textarea
-=============================
+# Simple Decorator for Textarea
```

Comment:
Wait i'm confused by this comment. Did you mean "original did not support textinput"?